tests/cpp/TestEtlRwExtensions.cpp

```
@@ -293,6 +298,26 @@ TEST(TestEtlRwExtensions, readBytearray)
 
+TEST(TestEtlRwExtensions, readEtlByte)
+{
+    etl::vector<uint8_t, 4> storage{0x11, 0xAB};
+    etl::byte_stream_reader reader(storage.begin(), storage.end(), etl::endian::little);
+
+    EXPECT_EQ(etl::byte(0x11), lrpc::read_unchecked<etl::byte>(reader));
+    EXPECT_EQ(etl::byte(0xAB), lrpc::read_unchecked<etl::byte>(reader));
+}
+
+#if (__cplusplus == 201703L)
+TEST(TestEtlRwExtensions, readStdByte)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)